ts/a11y/explorer/KeyExplorer.ts

```diff
@@ -248,11 +291,61 @@ export class SpeechExplorer extends AbstractExplorer<string> implements KeyExplo
     ['ArrowUp', (node: HTMLElement) => node.parentElement.closest(nav)],
     ['ArrowLeft', this.prevSibling.bind(this)],
     ['ArrowRight', this.nextSibling.bind(this)],
-    ['>', (_node: HTMLElement) => {
-      return null;
-    }],
+    ['>', this.nextRules.bind(this)],
+    ['<', this.nextStyle.bind(this)],
+    ['x', this.summary.bind(this)],
   ]);
 
+  /**
+   * Computes the summary for this expression. This is temporary and will be
+   * replaced by the full speech on focus out.
+   *
+   * @param {HTMLElement} node The targeted node.
+   * @return {HTMLElement} The refocused targeted node.
+   */
+  public summary(node: HTMLElement): HTMLElement {
+    this.generators.summary(node);
+    this.refocus(node);
+    return node;
+  }
+
+  /**
+   * Cycles to next speech rule set if possible and recomputes the speech for
+   * the expression.
+   *
+   * @param {HTMLElement} node The targeted node.
+   * @return {HTMLElement} The refocused targeted node.
+   */
+  public nextRules(node: HTMLElement): HTMLElement {
+    this.generators.nextRules(node);
+    this.Speech();
+    this.refocus(node);
+    return node;
+  }
+
+  /**
+   * Cycles to next speech style or preference if possible and recomputes the
+   * speech for the expression.
+   *
+   * @param {HTMLElement} node The targeted node.
+   * @return {HTMLElement} The refocused targeted node.
+   */
+  public nextStyle(node: HTMLElement): HTMLElement {
+    this.generators.nextStyle(node);
+    this.Speech();
+    this.refocus(node);
+    return node;
+  }
+
+  private refocus(node: HTMLElement) {
```

Comment:
Do you need jsdoc for this function and the one below?

Reply:
ditto